bin/snare

```diff
@@ -226,7 +230,8 @@ if __name__ == '__main__':
     loop = asyncio.get_event_loop()
     try:
         loop.run_until_complete(app.start())
-        drop_privileges()
+        if os.getuid() == 0:
```

Comment:
Sorry, I missed this part. I would prefer to drop privileges for all users, even without root permissions

Reply:
Hey @afeena 

Here's what the output is when I do it for a normal user:
```
>>> os.setgid(wanted_group.gr_gid)
PermissionError: [Errno 1] Operation not permitted

Can you please help me out here. I am new to using such a technique. Does a normal run (not superuser) require dropping privileges? Thanks and regards.

Reply:
My bad! Just keep you variant :)

Reply:
No biggie! Thanks a lot. :)